src/main/java/hudson/plugins/promoted_builds/conditions/GroovyConditionDescriptor.java

```diff
@@ -0,0 +1,47 @@
+package hudson.plugins.promoted_builds.conditions;
+
+import hudson.Extension;
+import hudson.PluginManager;
+import hudson.PluginWrapper;
+import hudson.model.AbstractProject;
+import hudson.plugins.promoted_builds.PromotionConditionDescriptor;
+import jenkins.model.Jenkins;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Declared outside GroovyCondition as it depends on the optional 'script-security' dependency
+ */
+@Extension
```

Comment:
Should be <code>@Extension(optional = true)</code> in order to avoid loading if there is no script-security plugin available. BTW I'm not sure it will work correctly, because the descriptor class does not depend on script-security itself